### PR TITLE
add_definitions

### PR DESCRIPTION
**Pull request by jaybee84**
### Purpose
Adds definitions to box 2

### Directions for reviewers
Due to a mistake on my end, the first part of the file got committed directly to the draft-branch. So the diff only shows changes in the later part of the file. Please disregard the diff and review the whole file for new definitions. Please feel free to add to the definitions as needed.

#### Which areas should receive a particularly close look?
definitions in box 2

#### Is there anything that you want to discuss further?
Are there other terms we should consider adding? 

#### Is the pull request ready for review?
yes

### Manuscript checklist

Unless otherwise noted above, this PR will be considered ready for review when all four items have been checked.

- [X] All changes to text follow "one sentence per line" [[Manubot instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#manubot-markdown)]
- [X] All citations follow the [Manubot citation instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#citations)
- [x] All changes or additions to tables follow the [Manubot table instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#tables)
- [ ] All figures follow the [Manubot figure instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#figures)
- [ ] All terms that need to be added to glossary have been added to #50 
- [ ] There are no new spelling errors identified by the Manubot spellchecker

### CODE REVIEW

**Comment by jaybee84** [comment]
pinging @jaclyn-taroni and @allaway for review :)

File: content/10. box_2_definitions.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ Transfer learning can be supervised (one or both of the source and target domain
 Regularization is an approach to reduce overfitting of models to training data, where a penalty or constraint is added to a model trained on a training dataset to avoid making large prediction errors on the evaluation dataset.
 
 ### Knowledge Graph
+Human knowledge about a domain can be abstracted into a form of network of nodes and edges collectively forming a knowledge graph. In this network any entity of interest (for example a gene, a disease, a protein, or a cell-line) can be represented as a node. All nodes can be linked in a directed manner through edges that represent known relationships between the nodes. For example a gene (node) can be linked to a protein (node) using a directed edge that represents the relationship that the protein is generated through the transcription and translation of the gene. Knowledge graphs serve to integrate data that exist is distributed sources, encode human readable knowledge in machine readable format, and evolve in a flexible manner to integrate new knowledge as it becomes available.
```
Comment (jaclyn-taroni) [inline content/10. box_2_definitions.md]: One sentence per line, and some other suggestions too. Don't love what I wrote about directed edges.

```suggestion
A knowledge graph is a network representation of human knowledge about a domain, abstracted into nodes and edges. 
Any entity of interest (for example a gene, a disease, a protein, or a cell-line) can be represented as a node in a knowledge graph.
All nodes can be linked through edges that represent known relationships between the nodes. 
Edges can be directed, indicating that the order of the nodes is important for encoding the relationship, or undirected.
For example, a gene (node) can be linked to a protein (node) using a directed edge that represents the relationship that the protein is generated through the transcription and translation of the gene. 
Knowledge graphs serve to integrate data that exist is distributed sources, encode human readable knowledge in machine readable format, and evolve in a flexible manner to integrate new knowledge as it becomes available.
```

File: content/10. box_2_definitions.md [hunk 2]
```diff
@@ -17,6 +17,7 @@ Transfer learning can be supervised (one or both of the source and target domain
...
 
 ### Rare Disease
-Rare diseases are defined
+According to the Orphan Drug Act[@https://www.fda.gov/media/99546/download] of United States of America, diseases or conditions that impact less than 200,000 people in the U.S are considered to be rare diseases. European Union defines a disease as rare when it affects less than 1 in 2,000 people.
```
Comment (jaclyn-taroni) [inline content/10. box_2_definitions.md]:
```suggestion
According to the Orphan Drug Act[@https://www.fda.gov/media/99546/download] of United States of America, diseases or conditions that impact less than 200,000 people in the U.S are considered to be rare diseases. 
The European Union defines a disease as rare when it affects less than 1 in 2,000 people.
```

**Review by jaclyn-taroni — COMMENTED** [review]
Suggesting one sentence per line + other edits for the knowledge graph section

File: content/10. box_2_definitions.md [hunk 3]
```diff
@@ -17,6 +17,13 @@ Transfer learning can be supervised (one or both of the source and target domain
 Regularization is an approach to reduce overfitting of models to training data, where a penalty or constraint is added to a model trained on a training dataset to avoid making large prediction errors on the evaluation dataset.
 
 ### Knowledge Graph
+A knowledge graph is a network representation of human knowledge about a domain, abstracted into nodes and edges. 
+Any entity of interest (for example a gene, a disease, a protein, or a cell-line) can be represented as a node in a knowledge graph.
+All nodes can be linked through edges that represent known relationships between the nodes. 
+Edges can be directed, indicating that the order of the nodes is important for encoding the relationship, or undirected.
+For example, a gene (node) can be linked to a protein (node) using a directed edge that represents the relationship that the protein is generated through the transcription and translation of the gene. 
+Knowledge graphs serve to integrate data that exist is distributed sources, encode human readable knowledge in machine readable format, and evolve in a flexible manner to integrate new knowledge as it becomes available.
```
Comment (jaybee84) [inline content/10. box_2_definitions.md]:
```suggestion
Knowledge graphs serve to integrate data that exist in distributed sources, encode human readable knowledge in machine readable format, and evolve in a flexible manner to integrate new knowledge as it becomes available.
```

**Comment by jaybee84** [comment]
Thanks @jaclyn-taroni !

**Pull request merged**